cl/alerts/templates/disable_docket_alert.html

```diff
@@ -0,0 +1,52 @@
+{% extends "base.html" %}
```

Comment:
This looks a lot better, thanks. I think we can now combine the disable and enable templates and just pass `enabled=True` or something similar into the template, right? 

They're so similar, it seems silly to repeat it.

Reply:
yeah, that's better, I've deleted disable_docket_alert.html and enable_docket_alert.html and merged them into a single template docket_alert.html, using a context variable (enabled) to change the content.